projects/ngx-skeleton-loader/src/lib/ngx-skeleton-loader.scss

```diff
@@ -114,6 +114,12 @@
   }
 }
 
+//For customContent mark height to 100% and remove the background
+.customContent {
```

Comment:
Sorry, mate! Looks like this code needs to be updated!

There's a new CS class used as namespace in place called `.skeleton-loader` https://github.com/willmendesneto/ngx-skeleton-loader/blob/main/projects/ngx-skeleton-loader/src/lib/ngx-skeleton-loader.scss#L1. So `.customContent` needs to live inside of this namespace.

Could you update this pull request with the latest changes, please? Also, it will be better for convention to use `custom-content` instead of `customContent` across the codebase